### PR TITLE
[FKS-15] : Configure UI framework

### PR DESCRIPTION
**Pull request by erfan-goodarzi**
Install and configure Mantine

### CODE REVIEW

**Comment by ASafaeirad** [comment]
Please fix the merge conflicts.

File: src/main.tsx [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <ColorSchemeProvider
+      colorScheme="light"
+      toggleColorScheme={() => {
+        console.log('');
+      }}
```
Comment (AmirabbasJ) [inline src/main.tsx:27]: can't we omit this field?
is it required?

File: src/main.tsx [hunk 2]
```diff
@@ -1,14 +1,43 @@
...
+    >
+      <MantineProvider
+        withGlobalStyles
+        withCSSVariables
+        emotionCache={rtlCache}
+        theme={{
+          dir: 'rtl',
+          fontFamily: 'Iranian Sans, sans-serif !important',
+          colorScheme: 'light',
```
Comment (AmirabbasJ) [inline src/main.tsx:36]: I think it's a better idea to have separate file for this
and why is `!important` there

**Review by AmirabbasJ — CHANGES_REQUESTED** [review]

File: src/main.tsx (hunk 1 above)
Comment (AmirabbasJ) [reply src/main.tsx:27]: then I'd say we remove the log and just make an empty function (maybe put a `NOTE` comment on it saying its required)
and if eslint has problem with empty function just ignore it with eslint ignore comment

Comment (AmirabbasJ) [reply src/main.tsx:27]: in this case, no

File: src/main.tsx [hunk 3]
```diff
@@ -1,14 +1,43 @@
...
 );
 
 root.render(
   <React.StrictMode>
-    <App />
+    <ColorSchemeProvider
+      colorScheme="light"
+      toggleColorScheme={() => {
+        // required
+      }}
```
Comment (ASafaeirad) [inline src/main.tsx:27]: We don't need dynamic color schemes in this app. Let's remove it.

**Review by ASafaeirad — CHANGES_REQUESTED** [review]

**Review by ASafaeirad — APPROVED** [review]

**Review by AmirabbasJ — APPROVED** [review]

**Pull request merged**